.github/CONTRIBUTING.md

```diff
@@ -482,16 +482,60 @@ in the SQL/updates folder.
   * Areas should not be var-edited on a map to change it's name or attributes. All areas of a single type and it's altered instances are considered the same area within the code, and editing their variables on a map can lead to issues with powernets and event subsystems which are difficult to debug.
 
+* If you are making non-minor edits to an area or room, (non-minor being anything more than moving a few objects or fixing small bugs) then you should ensure the entire area/room meets these standards.
+
+* When making a change to an area or room, follow these guidelines:
+  * Unless absolutely necessary, do not run pipes (including disposals) under wall turfs.
+  * NEVER run cables under wall turfs.
+  * Keep floor turf variations to a minimum. Generally, more than 3 floor turf types in one room is bad design.
+  * Run air pipes together where possible. The first example below is to be avoided, the second is optimal:
+  
+    ![image](https://user-images.githubusercontent.com/12197162/120011088-d22c7400-bfd5-11eb-867f-7b137ac5b1b2.png) ![image](https://user-images.githubusercontent.com/12197162/120011126-dfe1f980-bfd5-11eb-96b2-c83238a9cdcf.png)
+  * Pipe layouts should be logical and predictable, easy to understand at a glance. Always avoid complex layouts like in this example: 
+    
+    ![image](https://user-images.githubusercontent.com/12197162/120619480-ecda6f00-c453-11eb-9d9f-abf0d1a99c34.png)
+
+  * Decals are to be used sparingly. Good map design does not require warning tape around everything. Decal overuse contributes to maptick slowdown.
+  * Every **area** should contain only one APC and air alarm.
+    * Critical infrastructure rooms, such as the engine, should be given an APC with a larger power cell.
+  * Every **room** should contain at least one fire alarm, air vent, air scrubber, light switch, station intercom, and security camera.
+    * Intercoms should be set to frequency 145.9, and be speaker ON Microphone OFF. This is so radio signals can reach people even without headsets on. Larger room will require more than one at a time.
+    * Exceptions can be made to security camera placement for certain rooms, such as the execution room. Larger rooms may require more than one security camera. All security cameras should have a descriptive name that makes it easy to find on a camera console.
+    * Fire alarms should not be placed next to expected heat sources.
+    * Use the following "on" subtype of vents and scrubbers as opposed to var-editing: ```/obj/machinery/atmospherics/unary/vent_scrubber/on``` and ```/obj/machinery/atmospherics/unary/vent_pump/on```
```

Comment:
You might also want to mention the 'Mapping' verbs in this section, since they could probably help a lot with working some of this out. (Camera and Intercom range display especially)
![image](https://user-images.githubusercontent.com/57483089/120713153-6bb5c300-c4b9-11eb-9a67-0a1172825405.png)

Reply:
That could perhaps be something for a wiki mapping guide? Those tools _are_ useful, however, this is more of a set of standards as opposed to a "how-to guide".